arbos/l1pricing/l1pricing.go

```diff
@@ -68,12 +68,15 @@ const (
 )
 
 const (
-	InitialEquilibrationUnits uint64 = 60 * params.TxDataNonZeroGasEIP2028 * 100000 // one minute at 100000 bytes / sec
-	InitialInertia                   = 10
-	InitialPerUnitReward             = 10
-	InitialPricePerUnitWei           = 50 * params.GWei
+	InitialInertia           = 10
+	InitialPerUnitReward     = 10
+	InitialPricePerUnitWei   = 50 * params.GWei
+	InitialPerBatchGasCostV7 = 141000
```

Comment:
This should be set to the observed cost per batch, minus the `batchDataGas` value in a typical BatchPostingReport tx, which seems to be around 43000.  So I think 100000 is a better value.

Reply:
Here is an example BatchPostingReport tx: https://nova-explorer.arbitrum.io/tx/0x0c69c7ccead9c47a6d7ff70820b79a47a03b97037bae7116c1ba0b8f2a279532/internal-transactions

Reply:
Makes sense, let's lower this on Nova too (it's currently the `141000` I copied)